lib/active_merchant/billing/gateways/cyber_source_rest.rb

```diff
@@ -230,6 +230,72 @@ def add_merchant_description(post, options)
         merchant[:locality] = options[:merchant_descriptor_locality] if options[:merchant_descriptor_locality]
       end
 
+      def add_stored_credentials(post, payment, options)
+        stored_credential = options[:stored_credential]
+        if stored_credential
+          options = stored_credential_options(stored_credential, options.fetch(:reason_code, ''))
```

Comment:
Where does this `reson_code` come from? Are we planning to pass it to AM as part of the `stored_credential` options hash in addition to `reason_type`?